src/Predict/jobData.js

```diff
@@ -42,8 +42,8 @@ const jobCards = {
   polaris: {
     file: 'tiff_stack_examples/MERFISH_example.tiff',
     name: 'Polaris',
-    model: 'Polaris performs fluorescent spot detection.',
-    inputs: 'An image containing fluorescent spots generated by smFISH or similar assays. Pixel size of 0.1 μm recommended.',
+    model: 'Polaris performs fluorescent spot detection and can assign spots to segmented cells.',
+    inputs: 'An image containing fluorescent spots generated by smFISH or similar assays. For tissue segmentation, input an image containing both a nuclear marker and a membrane/cytoplasm marker. For cell culture segmentation, input an image containing nuclear and/or whole-cell label channels. Pixel size of 0.1 μm recommended.',
```

Comment:
How important is the difference between a membrane/cytoplasm marker and a whole-cell label? This could be rephrased as "To segment cells, include nuclear and whole-cell channels in the input image." 

If the difference is important, here's another rephrasing
"To segment tissue images, include a nuclear channel and a membrance/cytoplasm channel, or to segment cell culture, include a nuclear channel and/or a whole-cell channel."

Reply:
This is a good edit, thanks